packages/geojson-extension/package.json

```
@@ -34,8 +34,8 @@
     "watch": "tsc -w"
   },
   "dependencies": {
-    "@jupyterlab/apputils": "~2.0.0",
-    "@jupyterlab/rendermime-interfaces": "^2.0.0",
+    "@jupyterlab/apputils": "~2.1.0-a.0",
```

I think this may be pinned too tight by mistake, no other extension uses `~` but instead use `^`.

```suggestion
    "@jupyterlab/apputils": "^2.1.0",
```